app/src/main/java/com/haruta/harutyan/countchallenge/StartUpActivity.kt

```diff
@@ -0,0 +1,24 @@
+package com.haruta.harutyan.countchallenge
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.haruta.harutyan.countchallenge.databinding.ActivityStartUpBinding
+
+class StartUpActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityStartUpBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityStartUpBinding.inflate(layoutInflater).apply { setContentView(this.root) }
+
+        val countIntent: Intent = Intent(this, MainActivity::class.java)
```

Comment:
MainActivity に飛ぶので、 `mainIntent` とかでも良いかも！

```suggestion
        val mainIntent: Intent = Intent(this, MainActivity::class.java)
```